bot.toml

```diff
@@ -0,0 +1,25 @@
+[rpcs]
+l1-rpc = 
+l2-rpc = 
+
+[db]
+host = "0.0.0.0"
+port = 5434
+user = "db_username"
+password = "db_password"
+name = "db_name"
+
+[l1-contracts]
+address-manager                 = "0x1111111111111111111111111111111111111111"
```

Comment:
It's determined for both testnet and mainnet.

1. Remove unnecessary configurations.
2. List both testnet and mainnet configurations here. Use testnet as the default and comment out the mainnet configuration.